yotiprotoattr/Attribute.pb.go

```diff
@@ -5,8 +5,9 @@ package yotiprotoattr
 
 import (
 	fmt "fmt"
-	proto "github.com/golang/protobuf/proto"
 	math "math"
+
```

Comment:
Sorry, I should have noticed this before - since the protobuf files are all auto-generated, we have generally taken a decision to ignore them from all linting, since we'd have to update them again every time we regenerate, and it would add noise to the changeset every time.

Reply:
Oops. Not sure when this snuck in. I don't think there's anything I can do about this now.